ITE/4/README.adoc

```diff
@@ -570,6 +618,7 @@ This ITE currently proposes no prototypes.
 * link:https://github.com/cdfoundation/sig-security-sbom[CD Foundation Special Interest Group on Software Bill of Materials]
 * link:https://www.ntia.doc.gov/SoftwareTransparency[NTIA Software Component Transparency]
 * link:https://www.it-cisq.org/software-bill-of-materials/index.htm[CISQ/OMG Tool-to-tool Software Bill of Materials Exchange]
+* link:https://cnab.io/[Cloud Native Application Bundles]
```

Comment:
What about adding the other references to this section, too? For example the prototype implementation and the PR that was mentioned by @trishankatdatadog . I am a big fan of a big appendix of references :nerd_face:

Reply:
I've unfortunately ignored this PR a bit, sorry for the delay. I'll add links here, once we're sure about what goes into this PR. Thanks! :slightly_smiling_face: